cpp/ps0006/ps150days.cpp

```diff
@@ -109,7 +99,6 @@ int main()
     cout << "===============" << endl;
 
     counter(200, -44); // Massive number of iterations for our humble counter

```

Comment:
- [x] `start=0`, `step=1` as default
- [x] up counter works properly when `step` is positive
- [x] down counter works properly when `step` is negative
- [ ] counter gives error message for invalid value of `step`
- [x] up counter gives error message for negative value of  `step`  
- [x] down counter gives error message for positive value of  `step`
- [x] counter gives error message to count from a number to the same number unless `step==0`